feat: added init-private option (#8246)
npm/cli: docs/lib/content/commands/npm-init.md

$ npm init -y
```

> Set the private flag in package.json by default:

Reviewer: ```suggestion
Set the private flag to `true` in package.json:
```
